tests/jballerina-unit-test/src/test/resources/test-src/jvm/errors.bal

```diff
@@ -61,7 +61,7 @@ function testNestedCallsWithAllTraps(int y) {
 
 function validateError(int y, int|error x) {
     if (x is error) {
-        var expMessage = "reason foo " + y.toString();
+        string expMessage = "reason foo " + y.toString();
```

Comment:
Why do we need this change?

Reply:
We don't need this change. will revert this when addressing the remaining review suggestions 
Created https://github.com/ballerina-platform/ballerina-lang/issues/30313

Reply:
Fixed in 21f4087